frontend/src/App.css

```diff
@@ -140,13 +140,14 @@ body {
   font-size: 1.6;
   color: var(--color-gray-700);
 
-  overflow: hidden;
+  overflow-x: hidden;
   padding: 0;
   margin: 0;
-  height: 100vh;
+  /*removed the hieght attribute from here since it was cutting the upper half on addidng the footer*/
```

Comment:
I have made the change here so the UI doesnt gets clipped now it works perfectly pls check